docs/installation/kind-without-olm-linux.md

```diff
@@ -0,0 +1,182 @@
+
+Install the DevWorkspace Operator from the given URL:
+
+```sh
+kubectl apply -f https://github.com/devfile/devworkspace-operator/raw/refs/tags/v0.31.2/deploy/deployment/kubernetes/combined.yaml
+```
+
+Wait until the DevWorkspace Operator pods are ready:
+
+```sh
+kubectl wait --namespace devworkspace-controller \
+      --timeout 90s \
+      --for=condition=ready pod \
+      --selector=app.kubernetes.io/part-of=devworkspace-operator
+```
+
+### 6. Create the DevWorkspace Operator Config
+
+#### 6.1 Get Kind Node IP
+
+Get the internal IP address of your Kind control-plane node:
+
+```sh
+kubectl get node -o wide
+```
+Look for the `INTERNAL-IP` of the `kind-control-plane` node. Let's denote this as `<HOST_IP>`. You will use this IP in the next step.
+
+#### 6.2 Create the DevWorkspaceOperatorConfig
+
+Create the `DevWorkspaceOperatorConfig` resource, replacing `<HOST_IP>` with the IP you obtained in the previous step:
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: controller.devfile.io/v1alpha1
+kind: DevWorkspaceOperatorConfig
+metadata:
+  name: devworkspace-operator-config
+  namespace: devworkspace-controller
+config:
+  routing:
+    clusterHostSuffix: "<HOST_IP>.nip.io"
+EOF
+```
+
+### 7. Create a Sample DevWorkspace
+
+Create a sample DevWorkspace:
+
+```bash
+kubectl apply -f - <<EOF
+kind: DevWorkspace
+apiVersion: workspace.devfile.io/v1alpha2
+metadata:
+  name: git-clone-sample-devworkspace
+spec:
+  started: true
+  template:
+    projects:
+      - name: web-nodejs-sample
+        git:
+          remotes:
+            origin: "https://github.com/che-samples/web-nodejs-sample.git"
+      - name: devworkspace-operator
+        git:
+          checkoutFrom:
+            remote: origin
+            revision: 0.21.x
+          remotes:
+            origin: "https://github.com/devfile/devworkspace-operator.git"
+            amisevsk: "https://github.com/amisevsk/devworkspace-operator.git"
+    commands:
+      - id: say-hello
+        exec:
+          component: che-code-runtime-description
+          commandLine: echo "Hello from $(pwd)"
+          workingDir: ${PROJECT_SOURCE}/app
+  contributions:
+    - name: che-code
+      uri: https://eclipse-che.github.io/che-plugin-registry/main/v3/plugins/che-incubator/che-code/latest/devfile.yaml
+      components:
+        - name: che-code-runtime-description
+          container:
+            env:
+              - name: CODE_HOST
+                value: 0.0.0.0
+EOF
+```
+
+**Note:** The DevWorkspace creation may fail due to timeout if some container images are large and take longer than 5 minutes to pull. If the DevWorkspace fails, you can restart it by setting `spec.started` to `true`. Use the following command to re-trigger the DevWorkspace start:
+
+```bash
+kubectl patch devworkspace git-clone-sample-devworkspace -n default --type merge -p '{"spec": {"started": true}}'
+```
+You can also check the DevWorkspace status by running:
+```sh
+kubectl get devworkspace -n default
```

Comment:
WDYT about adding something like:

Reply:
When the DevWorkspace is running according to the status, open the editor by accesssing the URL from the `INFO` column in a web browser. For example:
```
NAME                            DEVWORKSPACE ID             PHASE     INFO
git-clone-sample-devworkspace   workspace0196ce197f0b4e90   Running   <URL>
```
?

Reply:
Yes, this is an important step I forgot to mention, and it should be in this doc.